### PR TITLE
Baseline sanity check

### PR DESCRIPTION
**Pull request by hantaowang**
Reruns the baseline after every iteration. If current test of baseline deviates from the original baseline by more than 10% (can be changed), it exits.

Also adds lumber sexual to the blacklists.

### CODE REVIEW

File: src/modify_resources.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 ip_pattern = r'\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b'
 
 # Container blacklist for container names whose placements should not move
-container_blacklist = ['ovn-controller', 'minion', 'ovs-vswitchd', 'ovsdb-server', 'etcd']
+container_blacklist = ['ovn-controller', 'minion', 'ovs-vswitchd', 'ovsdb-server', 'etcd', 'hantaowang/lumbersexual']
```
Comment (mchang6137) [inline src/modify_resources.py]: Note that we need to organize the blacklist in a centralized manner. For later PR.

File: src/test_config.cfg [hunk 2]
```diff
@@ -1,27 +1,31 @@
 [Basic]
 
-baseline_trials = 5
```
Comment (mchang6137) [inline src/test_config.cfg:3]: Can you add test_config.cfg to the .gitignore?

File: src/run_throttlebot.py [hunk 3]
```diff
@@ -751,6 +782,8 @@ def parse_config_file(config_file):
     sys_config['quilt_overhead'] = config.getint('Basic', 'quilt_overhead')
     sys_config['gradient_mode'] = config.get('Basic', 'gradient_mode')
     sys_config['setting_mode'] = config.get('Basic', 'setting_mode')
+    sys_config['rerun_baseline'] = config.getboolean('Basic', 'rerun_baseline')
```
Comment (mchang6137) [inline src/run_throttlebot.py:785]: Add documentation in README about what this parameter means.

File: src/modify_resources.py (hunk 1 above)
Comment (mchang6137) [inline src/modify_resources.py]: Should maintain separate blacklists: one for the Quilt blacklist and one for application specific services .

**Review by mchang6137 — CHANGES_REQUESTED** [review]

**Comment by hantaowang** [comment]
not sure how to remove test_config.cfg from merge but I have added it to the gitignore

**Review by mchang6137 — APPROVED** [review]

**Pull request merged**